dist/index.js

```diff
@@ -4372,6 +4372,7 @@ async function runAction() {
 			const lintDirRel = core.getInput(`${linterId}_dir`) || ".";
 			const prefix = core.getInput(`${linterId}_command_prefix`);
 			const lintDirAbs = join(context.workspace, lintDirRel);
+			const linterAutoFix = core.getInput(`${linterId}_auto_fix`) === "true";
```

Comment:
```suggestion
			const linterAutoFix = false === autoFix 
 ? false : core.getInput(`${linterId}_auto_fix`) === "true";
```

Can we set `linterAutoFix` false by default if `autoFix` is set to false? This would allow to replace the `autoFix && linterAutoFix` checks below with just `linterAutoFix`.